<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>original</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QGraphicsView" name="graphicsViewOriginal"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>border</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QGraphicsView" name="graphicsViewBorder"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Data</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Calibrate diametr(pt)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditVolume">
                <property name="inputMask">
                 <string>000</string>
                </property>
                <property name="text">
                 <string>270</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Calibrate diametr(mm)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditDiametr">
                <property name="inputMask">
                 <string>00.0000</string>
                </property>
                <property name="text">
                 <string>06.3500</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Weight (g)</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEditWeight">
                <property name="inputMask">
                 <string comment="00,0000">00.0000</string>
                </property>
                <property name="text">
                 <string>01.5025</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextEdit" name="textEditOut2"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="20,10,0,1">
        <item>
         <widget class="QGraphicsView" name="graphicsViewScaled"/>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditOut"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSave">
          <property name="text">
           <string>Save to text file</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Discret</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxDiscret">
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxInvert">
            <property name="text">
             <string>Invert</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMove">
    <property name="title">
     <string>Move</string>
    </property>
    <addaction name="actionMove_left"/>
    <addaction name="actionMove_right"/>
    <addaction name="actionMove_up"/>
    <addaction name="actionMove_down"/>
    <addaction name="actionTake_point"/>
    <addaction name="actionMoveF_left"/>
    <addaction name="actionMoveF_right"/>
    <addaction name="actionMoveF_up"/>
    <addaction name="actionMoveF_down"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="actionInterpolate"/>
    <addaction name="actionOpen_file_list"/>
    <addaction name="actionCalculate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMove"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMove_left">
   <property name="text">
    <string>move left</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionMove_right">
   <property name="text">
    <string>move right</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionMove_up">
   <property name="text">
    <string>move up</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionMove_down">
   <property name="text">
    <string>move down</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionTake_point">
   <property name="text">
    <string>take point</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionInterpolate">
   <property name="text">
    <string>Interpolate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionMoveF_left">
   <property name="text">
    <string>moveF left</string>
   </property>
   <property name="shortcut">
    <string>Shift+Left</string>
   </property>
  </action>
  <action name="actionMoveF_right">
   <property name="text">
    <string>moveF right</string>
   </property>
   <property name="shortcut">
    <string>Shift+Right</string>
   </property>
  </action>
  <action name="actionMoveF_up">
   <property name="text">
    <string>moveF up</string>
   </property>
   <property name="shortcut">
    <string>Shift+Up</string>
   </property>
  </action>
  <action name="actionMoveF_down">
   <property name="text">
    <string>moveF down</string>
   </property>
   <property name="shortcut">
    <string>Shift+Down</string>
   </property>
  </action>
  <action name="actionOpen_file_list">
   <property name="text">
    <string>Open file list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="text">
    <string>Calculate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
